static/js/server_events_dispatch.js

```diff
@@ -723,15 +723,6 @@ export function dispatch_normal_event(event) {
                 user_settings.presence_enabled = event.value;
                 $("#user_presence_enabled").prop("checked", user_settings.presence_enabled);
                 activity.redraw_user(page_params.user_id);
-
-                // Temporarily update the set of user_status.away_user_ids
```

Comment:
A small nit on this comment: I'd have phrased it as "Temporary transition code to update ..." -- to avoid ambiguity about whether it's temporary because some other code is going to update that field soon after.

Not particularly important, since it's deleted by the end of the PR :).